### PR TITLE
Date formatting

### PR DESCRIPTION
**Pull request by PyDario**
I used the code of @ChLah . Looks good to me now

### CODE REVIEW

File: Scenes/TitleScreen/TitleScreen.gd [hunk 1]
```diff
@@ -5,7 +5,7 @@ func _ready():
 	if OS.get_name() == 'HTML5':
 		get_node("MainMenu/HBoxContainer/VBoxContainer/VBoxContainer/ExitLabel").set_visible(false)
 	$MainMenu/HBoxContainer/VBoxContainer/VBoxContainer/NewGameLabel.grab_focus()
-
+
```
Comment (ChLah) [inline Scenes/TitleScreen/TitleScreen.gd]: This change is unnecessary

File: Scenes/Highscore/Highscore.gd [hunk 2]
```diff
@@ -11,7 +11,40 @@ func _on_HTTPRequest_request_completed(result, response_code, headers, body):
 	var highscoretext = ""
 	var num = 1
 	for entry in json.dreamlo.leaderboard.entry:
-		highscoretext += "%3s: %40s %50s %20s\n" % [num, entry.name, entry.date, entry.score]
+
+		var date = entry.date
+		var splitup = date.split(" ", false, 0)
+		if splitup.size() > 3:
```
Comment (ChLah) [inline Scenes/Highscore/Highscore.gd]: What if size() < 3? Then you would later access elements in the array, that do not exist.
Please change to != 3

Comment (ChLah) [reply Scenes/Highscore/Highscore.gd]:
```suggestion
		if splitup.size() != 3:
```

File: Scenes/Highscore/Highscore.gd [hunk 3]
```diff
@@ -11,7 +11,40 @@ func _on_HTTPRequest_request_completed(result, response_code, headers, body):
...
+			print("Error: splitup too big")
```
Comment (ChLah) [inline Scenes/Highscore/Highscore.gd]: You can use push_error

```suggestion
			push_error("Error: Wrong format")
```

**Review by ChLah — CHANGES_REQUESTED** [review]
Can be improved.
My comment on the first check surely applies to the other checks too

File: Scenes/Highscore/Highscore.gd (hunk 3 above)
Comment (PyDario) [reply Scenes/Highscore/Highscore.gd]: Applied your changes.

**Comment by ChLah** [comment]
Looks okay now. Gonna merge it to master and then merge this with my changes from #29.

**Pull request merged**